### ISSUE TITLE
Animation on bounds change

### ISSUE THREAD

**Issue by cooksimo** [kolinkrewinkel/KKGridView#139]
Hi,

Is there a way to have the grid animate when laying itself out when the bounds of the gridview change?  I have a grid in a resizeable pane in an iPad app and when dragging it such that a column would be added/removed it just appears.  I would like it to animate in the same way it animates when rotating.

Thanks!

**Comment by kolinkrewinkel** [comment]
Good idea: tough to execute.  I'll definitely think about how to go about doing this..
